database/database.go

```diff
@@ -4,6 +4,7 @@ import (
 	"database/sql"
 	"io"
 
+	// TODO: Why do we import mysql here?
```

Comment:
It's required as a plugin for database/sql

Reply:
Okay.